src/components/common/EthHashInfo/index.tsx

```diff
@@ -70,10 +70,12 @@ const EthHashInfo = ({
         )}
 
         <Box className={css.addressRow}>
-          <Typography variant="body2" fontWeight="inherit" component="div" className={css.address}>
+          <Typography variant="body2">
```

Comment:
This doesn't inherit the `font-weight` anymore.

Reply:
It was repeated in the `.address` css